server/server.go

```diff
@@ -46,16 +48,23 @@ type Config struct {
 	GRPCStreamMiddleware []grpc.StreamServerInterceptor `yaml:"-"`
 	HTTPMiddleware       []middleware.Interface         `yaml:"-"`
 
-	GPRCServerMaxRecvMsgSize       int  `yaml:"grpc_server_max_recv_msg_size"`
-	GRPCServerMaxSendMsgSize       int  `yaml:"grpc_server_max_send_msg_size"`
-	GPRCServerMaxConcurrentStreams uint `yaml:"grpc_server_max_concurrent_streams"`
+	GPRCServerMaxRecvMsgSize        int           `yaml:"grpc_server_max_recv_msg_size"`
+	GRPCServerMaxSendMsgSize        int           `yaml:"grpc_server_max_send_msg_size"`
+	GPRCServerMaxConcurrentStreams  uint          `yaml:"grpc_server_max_concurrent_streams"`
+	GRPCServerMaxConnectionIdle     time.Duration `yaml:"grpc_max_connection_idle"`
```

Comment:
To keep consistency with the existing yaml options, maybe it's better if prefix the YAML fields with `grpc_server_`  instead of just `grpc_`.